src/main/java/org/jitsi/jigasi/transcription/Transcriber.java

```diff
@@ -200,43 +193,44 @@ public void add(String name, String id, long ssrc)
     public void remove(String name, String id, long ssrc)
     {
         transcript.notifyLeft(name);
-        if(this.participants.containsKey(ssrc))
+        if (this.participants.containsKey(ssrc))
         {
             participants.get(ssrc).left();
-            logger.debug("Removed participant "+name+" with ssrc "+ssrc);
+            logger.debug("Removed participant " + name + " with ssrc " + ssrc);
             return;
         }
 
         logger.warn("Asked to remove participant" + name + " with ssrc " +
-                ssrc + " which did not exist");
+                        ssrc + " which did not exist");
     }
 
     /**
      * Start transcribing all participants added to the list
      */
     public void start()
     {
-        if(State.NOT_STARTED.equals(this.state))
+        if (State.NOT_STARTED.equals(this.state))
         {
             logger.debug("Transcriber is now transcribing");
             this.state = State.TRANSCRIBING;
             this.executorService = Executors.newSingleThreadExecutor();
 
             List<String> names = new ArrayList<>(participants.size());
-            participants.values().forEach((p) -> {
-                String name;
-                if((name = p.getName()) == null)
-                {
-                    name = p.getId();
-                }
-                names.add(name);
-            });
+            participants.values().forEach((p) ->
+                                          {
+                                              String name;
+                                              if ((name = p.getName()) == null)
```

Comment:
Fixed (which also removed the weird indentation, it was introduced by my IDE and I failed to notice)